docs/http/http-spans.md

```diff
@@ -295,52 +298,58 @@ See the examples for more details about:
 
 ## HTTP server
 
-To understand the attributes defined in this section, it is helpful to read the "Definitions" subsection.
+Read the following section to understand how HTTP server instrumentations capture server information.
```

Comment:
```suggestion
Read the following section to understand how HTTP server instrumentations should capture server information.
```

Reply:
should is normative and must be capitalized, FYI

Reply:
this part is informational and non-normative. There is a normative section ("Setting `server.address` and `server.port` attributes") where normative verbs are capitalized.

Reply:
I know this is awkward, but we should avoid any normative verb in this section of docs.